ciao-controller/command.go

```diff
@@ -247,6 +255,7 @@ func (c *controller) launchCNCI(tenantID string) error {
 		WorkloadID: workloadID,
 		TenantID:   tenantID,
 		Instances:  1,
+		Name:       "cnci",
```

Comment:
As names are optional I can avoid this problem by just not setting a name. The CNCI already gets a special hostname in the code anyway.